core/templates/pages/skill-editor-page/editor-tab/skill-prerequisite-skills-editor/skill-prerequisite-skills-editor.directive.ts

```diff
@@ -53,7 +53,8 @@ angular.module('oppia').directive('skillPrerequisiteSkillsEditor', [
           var ctrl = this;
           var categorizedSkills = null;
           var untriagedSkillSummaries = null;
-          TopicsAndSkillsDashboardBackendApiService.fetchDashboardData().then(
+          TopicsAndSkillsDashboardBackendApiService.fetchDashboardDataAsync()
+            .then(
```

Comment:
Move the `function(response) {` from the line below, so it will be `.then(function(response) {`

Reply:
ditto in the other places

Reply:
Updated